tests/queries/query_test.py

```diff
@@ -11,19 +11,21 @@
 def run_client(bin_prefix, port, query, reference, replace_map={}):
     client = subprocess.Popen([bin_prefix + '-client', '--port', str(port), '-m', '-n', '--testmode'],
                               stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    result, error = client.communicate(query)
+    result, error = client.communicate(query.encode('utf-8'))
```

Comment:
Maybe it will be better to use `universal_newlines=True` for `Popen` ?

Reply:
No, because we have tests with binary references.